app/models/friendship.rb

```diff
@@ -1,2 +1,4 @@
 class Friendship < ApplicationRecord
+  belongs_to :request_sender, class_name: 'User'
+  belongs_to :request_receiver, class_name: 'User'
```

Comment:
#### This should be edited in order to meet with the user changes

- [ ] Your user model should have `belong_to :user` since this state that friendships will belong to the user
- [ ] Your user model should have `belong_to :friend` with a class pointed to the user to ensure that we'll link the user table thought friend_id

> check out [this link](https://smartfunnycool.com/friendships-in-activerecord/) for more info